pkg/cmd/auth/status/status_test.go

```diff
@@ -340,9 +339,13 @@ func Test_statusRun(t *testing.T) {
 			} else {
 				assert.NoError(t, err)
 			}
-
-			output := strings.ReplaceAll(stderr.String(), config.ConfigDir()+string(filepath.Separator), "GH_CONFIG_DIR/")
-			assert.Equal(t, tt.wantOut, output)
+			output := strings.ReplaceAll(stdout.String(), config.ConfigDir()+string(filepath.Separator), "GH_CONFIG_DIR/")
+			errorOutput := strings.ReplaceAll(stderr.String(), config.ConfigDir()+string(filepath.Separator), "GH_CONFIG_DIR/")
+			if output != "" {
+				assert.Equal(t, tt.wantOut, output)
```

Comment:
I'd rather see the tests changed to have a `wantErrOut` property that is compared to errOutput instead of conditionally checking stdout vs. stderr.

Reply:
Thanks for the review @vilmibm . I made the suggested change.